plugins/scanners/scancode/src/test/assets/scancode-3.0.2_mime-types-2.1.18.json

```diff
@@ -17,7 +17,7 @@
         "--strip-root": true,
         "--timeout": "300.0"
       },
-      "notice": "Generated with ScanCode and provided on an \"AS IS\" BASIS, WITHOUT WARRANTIES\nOR CONDITIONS OF ANY KIND, either express or implied. No content created from\nScanCode should be considered or used as legal advice. Consult an Attorney\nfor any legal advice.\nScanCode is a free software code scanning tool from nexB Inc. and others.\nVisit https://github.com/nexB/scancode-toolkit/ for support and download.",
+      "notice": "Generated with ScanCode and provided on an \"AS IS\" BASIS, WITHOUT WARRANTIES\nOR CONDITIONS OF ANY KIND, either express or implied. No content created from\nScanCode should be considered or used as legal advice. Consult an Attorney\nfor any legal advice.\nScanCode is a free software code scanning tool from aboutcode-org Inc. and others.\nVisit https://github.com/aboutcode-org/scancode-toolkit/ for support and download.",
```

Comment:
I guess "nexB Inc." should stay as-is here, as "aboutcode-org" is not an "Inc.", or?

Reply:
And actually, at the time this file was created, this was the correct output from ScanCode, so I believe the file should stay as-is.

Similar for all other JSON result files below. IMO these should not be touched at all.

Reply:
You're absolutely correct. Revert the reference links for all the test json files.

Reply:
Thanks. Please squash your second commit into the first one, as our final Git history should not contain commits of fixes that were discovered during PR review.